src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDays = 3;
+  const weekDays = 7;
```

Comment:
these names do not fully refer to what the variables contain, take care of it next times